test/app.test.ts

```diff
@@ -289,6 +289,23 @@ test('apps with varying yamlOutputTypes; two charts, no objects', () => {
   }
 });
 
+test('return app as yaml string', () => {
+  // GIVEN
+  const app = Testing.app();
+
+  // WHEN
+  const chart = new Chart(app, 'chart1');
+  new Chart(app, 'chart2');
+  new ApiObject(chart, 'obj1', { apiVersion: 'v1', kind: 'Kind1' });
+  new ApiObject(chart, 'obj2', { apiVersion: 'v1', kind: 'Kind2' });
+
```

Comment:
`chart2` covers the empty case but let's also add `chart3` with an API object to make sure the result includes objects from multiple charts.